graphsdk/src/androidTest/java/com/microsoft/graph/concurrency/SynchronousExecutorTests.java

```diff
@@ -22,6 +22,5 @@ public void run() {
         });
         simpleWaiter.waitForSignal();
         assertTrue(success.get());
-        assertEquals(0, synchronousExecutor.getActiveCount());
```

Comment:
@TBag Why was this check removed, if we are getting back a non-zero amount this should be a bug, no?